packages/rnv/src/platformTools/bundler.js

```diff
@@ -5,43 +5,37 @@ import Config from '../config';
 
 export const isBundlerRunning = async (c) => {
     try {
-        const { data } = await axios.get(`http://127.0.0.1:${c.runtime.port}`);
-        if (data.includes('React Native')) return true;
+        const { data } = await axios.get(`http://127.0.0.1:${c.runtime.port}/index.ios.bundle`);
```

Comment:
is this intentional? Showing loading of ios bundle while the original command was for android can confuse users

Reply:
true that, didn't think of it. Will change to reflect platform

Reply:
@mihaiblaga89 android is fixed. ios however will get this:

```
runtime com.apple.CoreSimulator.SimRuntime.iOS-13-3
⚠️  $ rnv run -p ios -t - WARNING: Your specified simulator target true doesn't exists
? Select the simulator you want to launch (Use arrow keys)
❯ iPhone 8 | Phone 📱 | v: iOS-13-3 | udid: E475674B-CCCE-4E4E-B34B-D00FEAE6CC5E
  iPhone 8 Plus | Phone 📱 | v: iOS-13-3 | udid: 676C7C32-B16E-41B6-BF0E-C71429EE9639
  iPhone 11 | Phone 📱 | v: iOS-13-3 | udid: C80138FC-7E80-413F-8511-6081E1405AFB
  iPhone 11 Pro | Phone 📱 | v: iOS-13-3 | udid: 11225487-D2C8-47E2-AE5C-74D9C1C33FB5
  iPhone 11 Pro Max | Phone 📱 | v: iOS-13-3 | udid: E745AE34-941A-4B07-B621-835EDF2D8022
  iPad Pro (9.7-inch) | Tablet 💊 | v: iOS-13-3 | udid: 5DE8C58A-A207-4820-8B5E-FC747BF89712
  iPad (7th generation) | Tablet 💊 | v: iOS-13-3 | udid: EDBFC801-D83B-41DD-A3E4-BA5CE980316C
Loading dependency graph, done.
```

if you run `rnv run -p ios -t`

however I'll merge this fix. there is another branch which will tackles ios simulator related issues